File: trunk/FarNet/Plugins.NET/Hello.CS/bin/Hello.cs
using FarManager;
public class HelloCS : BasePlugin
{
	void item_OnOpen(object sender, OpenPluginMenuItemEventArgs e)
	{
		Far.Msg("Hello, world from c#", "Far.NET");
	}
	void sayHello(string s)
	{
		Far.Msg("C#: Hello, " + s);
	}
	override public void Connect()
	{
		IPluginMenuItem menuItem = Far.CreatePluginsMenuItem();
		menuItem.Name = "Hello c#";
		menuItem.OnOpen += item_OnOpen;
		Far.RegisterPluginsMenuItem(menuItem);
		Far.RegisterPrefix("hellocs", new StringDelegate(sayHello));
	}
}

File: FarNet/Plugins.NET/Hello.JS/bin/Hello.js
import FarManager;
public class HelloJS extends BasePlugin{
	function item_OnOpen(sender:Object, e:OpenPluginMenuItemEventArgs) {
		Far.Msg("Hello, world from js.net");
	}
	function sayHello(s:String){
		Far.Msg("(JS.Net)Hello, "+s);
	}
	function Connect(){
		Far.RegisterPluginsMenuItem("Hello js.net", this.item_OnOpen);
		Far.RegisterPrefix("hello", this.sayHello);
	}
}
